public/views/error.template.html

```diff
@@ -10,7 +10,7 @@
 
     <base href="[[.AppSubUrl]]/" />
 
-    <link rel="stylesheet" href="public/build/grafana.[[ .Theme ]].css?v[[ .BuildVersion ]]">
+    <link rel="stylesheet" href="public/build/grafana.[[ .Theme ]].<%= webpack.hash %>.css">
```

Comment:
Nice! But how does this work, what is webpack.hash a a hash of since there is no filename in that context, a hash of the whole webpack run?

Reply:
Hash is a "build-specific hash" [1] which seems to be more like a unique build id. All the js files use the same hash feature [2]

[1] https://webpack.js.org/guides/caching/#output-filenames
[2] https://github.com/grafana/grafana/blob/d0776937b5e854c532574a9e7233b944a1957626/scripts/webpack/webpack.common.js#L14